src/browser/api/window.js

```diff
@@ -502,6 +502,25 @@ Window.create = function(id, opts) {
             }
         });
 
+        var emitToAppAndWin = (...types) => {
+            types.forEach(type => {
+                ofEvents.emit(route.window(type, uuid, name), { topic: 'window', type, uuid, name });
+                ofEvents.emit(route.application(type, uuid), { topic: 'application', type, uuid });
```

Comment:
Lets add the window name here, in an app that has multi origin windows we will need the window name to be able to determine what crashed.